docs/boshdirector.adoc

```diff
@@ -1,20 +1,32 @@
-= Installing Bosh Director
+= Configuring Ops Manager Director
 Eitan Suez <esuez@pivotal.io>
 v0.1, 2017
 :project_id: {{project_id}}
 :domain_name: {{domain_name}}
 :env_name: {{env_name}}
 :director_ip_address: {{director_ip_address}}
+:region: {{region}}
 
+Estimated time:  30 minutes
+
+= Requirements
+
+Lab Requirements
+
+= What you will learn
+
+* How to configure and install Ops Manager Director
```

Comment:
i likewise find these 'what you will learn sections' shallow and relatively uninformative.  doesn't the title of this document make clear that we're installing bosh?